grid2op/utils/l2rpn_idf_2023_scores.py

```diff
@@ -152,33 +162,34 @@ def _compute_episode_score(
         )
         # should match underlying_statistics.run_env `dict_kwg["other_rewards"][XXX] = ...`
         # XXX is right now f"{EpisodeStatistics.KEY_SCORE}_{nm}" [this should match the XXX]
-        
-        #retrieve nres_score
+
+        # retrieve nres_score
         new_renewable_sources_usage_score_nm = "new_renewable_sources_usage_scores"
         real_nm = EpisodeStatistics._nm_score_from_attr_name(new_renewable_sources_usage_score_nm)
         key_score_file = f"{EpisodeStatistics.KEY_SCORE}_{real_nm}"
         nres_score = float(other_rewards[-1][key_score_file])
         nres_score = max(nres_score, self.min_nres_score / self.scale_nres_score)
-        nres_score = self.scale_nres_score * nres_score 
-        
-        #assistant_confidence_score
-        # new_renewable_sources_usage_score_nm = "assistant_confidence_scores"
-        # real_nm = EpisodeStatistics._nm_score_from_attr_name(new_renewable_sources_usage_score_nm)
-        # key_score_file = f"{EpisodeStatistics.KEY_SCORE}_{real_nm}"
-        # assistant_confidence_score = float(other_rewards[-1][key_score_file])
-        assistant_confidence_score = 0 #self.scale_assistant_score * assistant_confidence_score
-        
-        #assistant_cost_score
-        # new_renewable_sources_usage_score_nm = "assistant_cost_scores"
-        # real_nm = EpisodeStatistics._nm_score_from_attr_name(new_renewable_sources_usage_score_nm)
+        nres_score = self.scale_nres_score * nres_score
+
+        # assistant_confidence_score
+        assistant_confidence_score_nm = "assistant_confidence_scores"
+        real_nm = EpisodeStatistics._nm_score_from_attr_name(assistant_confidence_score_nm)
+        key_score_file = f"{EpisodeStatistics.KEY_SCORE}_{real_nm}"
+        assistant_confidence_score = float(other_rewards[-1][key_score_file])
+        assistant_score = self.scale_assistant_score * assistant_confidence_score
+
+        # assistant_cost_score
+        # assistant_cost_score_nm = "assistant_cost_scores"
+        # real_nm = EpisodeStatistics._nm_score_from_attr_name(assistant_cost_score_nm)
         # key_score_file = f"{EpisodeStatistics.KEY_SCORE}_{real_nm}"
         # assistant_cost_score = float(other_rewards[-1][key_score_file])
-        assistant_cost_score  = 0 #self.scale_assistant_score * assistant_cost_score
-        
-        assistant_score = self.weight_confidence_assistant_score * assistant_confidence_score +\
-             (1. - self.weight_confidence_assistant_score) * assistant_cost_score
+        # assistant_cost_score = max(assistant_cost_score, self.min_assistant_cost_score / self.scale_assistant_score)
+        # assistant_cost_score  = self.scale_assistant_score * assistant_cost_score
+
+        # assistant_score = self.weight_confidence_assistant_score * assistant_confidence_score +\
+        #      (1. - self.weight_confidence_assistant_score) * assistant_cost_score
```

Comment:
A lot of this has been commented, we should maybe juste remove it (we no longer have two components in the assistant score).